CHANGES

```diff
@@ -33,7 +33,8 @@ Features added
 --------------
 
 * #4826: py domain: Add ``:canonical:`` option to python directives to describe
-  the location where the object is defined
+  the location where the object is defined.  This option is automatically added
+  by :mod:`sphinx.ext.autodoc` when relevant.
```

Comment:
This is an entry for the python domain, not for autodoc. Please add another entry. I think it would be better to describe how the autodoc extension is improved (not how implemented).